From be2b05b4391044607cd38838b379b16fd14992fa Mon Sep 17 00:00:00 2001
From: Harry Wentland <harry.wentland@amd.com>
Date: Fri, 3 Mar 2017 10:08:59 -0500
Subject: [PATCH 0698/2094] Revert "drm/amd/display: Enable warnings as errors"

This won't work in staging build where everyone uses different
gcc versions.

This reverts commit 8f1792d538f1781bba262be7eefad159d74eb51f.
---
 drivers/gpu/drm/amd/display/Makefile | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/gpu/drm/amd/display/Makefile b/drivers/gpu/drm/amd/display/Makefile
index c22475b..8ba37dd 100644
--- a/drivers/gpu/drm/amd/display/Makefile
+++ b/drivers/gpu/drm/amd/display/Makefile
@@ -5,8 +5,6 @@
 
 AMDDALPATH = $(RELATIVE_AMD_DISPLAY_PATH)
 
-subdir-ccflags-y += -Werror
-
 subdir-ccflags-y += -I$(AMDDALPATH)/ -I$(AMDDALPATH)/include
 
 subdir-ccflags-y += -I$(FULL_AMD_DISPLAY_PATH)/dc/inc/
-- 
2.7.4

